### ISSUE TITLE
ntfy plugin usage

### ISSUE THREAD

**Issue by webdb22** [josephdadams/CroniclePlugins#3]
First of all big thanks for sharing this brilliant plugins!

Installing the ntfy plugin was quite straight forward and it works fine being used standalone. But I can't figure out how to hand over information from the calling event via Chain Reaction to the event based on the ntfy plugin to set the parameters for the ntfy message to be sent.

Any details on the usage or a code snippet for what to put in the calling event as well as in the being called event (ntfy) would be highly appreciated, thanks a lot in advance!

**Comment by josephdadams** [comment]
The first event’s plugin has to support chaining.

here’s one that does it: https://github.com/josephdadams/CroniclePlugins/blob/master/ajakipro_checkavailablespace.py

Look at the end, when it fails, it prints the chain data. Then on that event I have error chaining set up to send it to the ntfy event.

**Comment by webdb22** [comment]
Hi Joseph,

thanks for your prompt reply, highly appreciated!

The example given helps a lot for the source event, so you just print / echo the data structure to be handed over, correct? 

What about the target, i.e. the event called via Chain Reaction, how to map the data to input parameters to e.g. set the title for the ntfy message?

**Comment by josephdadams** [comment]
https://github.com/jhuckaby/Cronicle/blob/master/docs/Plugins.md#chain-reaction-control

**Comment by webdb22** [comment]
@josephdadams Thanks a lot for your  hints, much appreciated! The good news is that after a lot of trial and error I got the basic stuff working! 

Off topic: Just to let you know, since I'll probably be moving to [ntfy](https://ntfy.sh/) for all types of notifications, I discovered [Mailrise](https://github.com/YoRyan/mailrise), which in turn uses [Apprise](https://github.com/caronc/apprise), to convert emails from applications that have no other notification capability to basically any type of notification, including ntfy.
